### PR TITLE
import_data testing and refactoring

### PR DESCRIPTION
**Pull request by richaagarwal**
This is the first set of tests for testing the management commands. It involves light refactoring and clean-up. More tests are coming, but I'd like to merge in frequently. 

## Additions

- tests

## Removals

- duplicate files/ unnecessary code

## Changes

- some refactoring to enable writing tests

## Testing

- run `tox`

## Review

- @kave @rosskarchner @kave 

## Notes

- open to suggestions on how to test the `convert` methods, as right now I'm just feeding it a giant dictionary and expecting the output it currently gives out. (also open to suggestions on where to put those dictionaries).
- Since I renamed import-data to import_data, the diff in Files Changed won't be useful, so refer to this commit to see the changes instead: https://github.com/cfpb/cfgov-refresh/commit/e652454634c07c81ecc467e81412185d784c9a96

## Todos

- still more to do, but this part is ready to be merged in. since there's some refactoring/renaming, I wanted to get this in sooner than later. 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/jinja2/v1/_lib/django_calendar_event_processor.py [hunk 1]
```diff
@@ -1,46 +0,0 @@
-import sys
```
Comment (kave) [inline cfgov/jinja2/v1/_lib/django_calendar_event_processor.py]: i see these have been removed, are they no longer necessary? i thought they were being used for the sheer_indexing

**Comment by kave** [comment]
i’m also getting the `sheer_index` error locally when i try to do so from your branch. could you look into it?

File: cfgov/jinja2/v1/_lib/django_calendar_event_processor.py (hunk 1 above)
Comment (richaagarwal) [inline cfgov/jinja2/v1/_lib/django_calendar_event_processor.py]: Yeah, don't think they're necessary. Thanks for pointing out the sheer_indexing issue though - fixing that now!

**Comment by kurtw** [comment]
> i’m also getting the `sheer_index` error

@kave that's due to [this](https://github.com/cfpb/owning-a-home/blob/master/src/_lib/wordpress_journey_processor.py#L23) expecting the `name` argument. @richaagarwal I'd suggest either making a PR on that repo or adding `name` back into the passed args.

**Comment by kave** [comment]
looks great, but i'm still having indexing errors on this branch. Could you verify that `name` is still not required?
```
indexed 16 office 
indexed 2 views 
indexed 65 sub_page 
indexed 4 faq 
indexed 49 orgmember 
Traceback (most recent call last):
  File "./cfgov/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/apaue/.virtualenvs/v1/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/Users/apaue/.virtualenvs/v1/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/apaue/.virtualenvs/v1/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/apaue/.virtualenvs/v1/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/Users/apaue/workspace/f5/cfgov-refresh/cfgov/sheerlike/management/commands/sheer_index.py", line 41, in handle
    index(args, options)
  File "/Users/apaue/workspace/f5/cfgov-refresh/cfgov/sheerlike/indexer.py", line 164, in index
    reindex=options.get('reindex'))
  File "/Users/apaue/workspace/f5/cfgov-refresh/cfgov/sheerlike/indexer.py", line 80, in index_processor
    document_iterator = processor.documents()
  File "/Users/apaue/workspace/f5/cfgov-refresh/cfgov/sheerlike/indexer.py", line 37, in documents
    return self.processor_module.documents(**self.kwargs)
TypeError: documents() takes exactly 2 arguments (1 given)
```

File: cfgov/v1/processors/eventpage.py [hunk 2]
```diff
@@ -5,98 +5,127 @@
...
+        else:
+            times_dict['date_published'] = datetime.date.today()
+        for t in times:
+            if t[0]:
+                dt = dateutil.parser.parse(t[0].get('date'))
+                times_dict[t[1]] = dt.strftime('%Y-%m-%d %H:%M')
+        return times_dict
+
+    def convert(self, doc):
+        post_dict = {
```
Comment (kave) [inline cfgov/v1/processors/eventpage.py]: noice much cleaner :+1:

**Comment by kurtw** [comment]
So that's an issue with the tag data conflicting. One solution would be to change the name of the tag names to the name in the import. For example, change `Financial empowerment` to `Financial Empowerment` then run the import.

**Comment by kurtw** [comment]
Runs fine for me though!

**Comment by richaagarwal** [comment]
@kurtw I'm not sure I follow - do you mean change the tag name in Wordpress? Do we still want to leave this try/catch in there though? @kave I'm going to deal with the sheer indexing issue by submitting a PR (once I'm able to, can't seem to get the permissions working) to Owning a Home.  

If things look good otherwise, I'd suggest we merge this in. Thanks!

**Comment by richaagarwal** [comment]
I undid the removal of the `name` param from `def documents` since there seems to be too much impacted code outside this repository.  @kave could you confirm sheer_index works for you?

**Comment by kurtw** [comment]
I mean change the tag in the MySQL database.

**Comment by kave** [comment]
:+1: everything works now.

**Comment by richaagarwal** [comment]
@kurtw Okay, so should I remove my check for the IntegrityError or is it OK to leave that in?

**Pull request merged**